core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -51,14 +51,14 @@ require(
 angular.module('oppia').component('contributorDashboardPage', {
   template: require('./contributor-dashboard-page.component.html'),
   controller: [
-    '$rootScope', 'LanguageUtilService', 'LocalStorageService',
+    '$rootScope', '$window', 'LanguageUtilService', 'LocalStorageService',
```

Comment:
Actually, can we just use WindowRef here and drop `$window` entirely?

Reply:
Done